tstyche.config.json

```diff
@@ -1,8 +1,8 @@
 {
+  "$schema": "https://tstyche.org/schemas/config.json",
+  "checkSuppressedErrors": true,
+  "checkSourceFiles": false,
```

Comment:
Only test files are checked for errors. This options enables type checking for all other files in the program (useful when `.d.ts` is handcrafted).

In your case all `.ts` files (including `docs`) belong to one project, a single `tsconfig.json`. So I think, disabling is a better idea. Otherwise it does too much.